test/sqliteTestApp/repos.gradle

```diff
@@ -0,0 +1,16 @@
+// Custom Repos
+
+ext.repos = {
+    google()
+    mavenCentral()
+    maven {
+        url = uri("https://archiva.totalpave.com/repository/internal")
+        credentials {
+            username = project.findProperty("totalpave.repository.user").toString()
+            password = project.findProperty("totalpave.repository.password").toString()
+        }
+    }
```

Comment:
Because this is a public repo it should not use any internals therefore it doesn't need the internal repo.